config/accelerate_fsdp_llama_2_procs.yaml

```diff
@@ -0,0 +1,27 @@
+# options that can be used with accelerate config are neatly documented here - 
+# https://github.com/huggingface/accelerate/blob/ee163b66fb7848892519e804688cb4ae981aacbe/docs/source/package_reference/cli.md
+
+compute_environment: LOCAL_MACHINE
+debug: false
+distributed_type: FSDP
+fsdp_config:
+  fsdp_auto_wrap_policy: TRANSFORMER_BASED_WRAP
+  fsdp_backward_prefetch_policy: BACKWARD_PRE
+  fsdp_forward_prefetch: true
+  fsdp_cpu_ram_efficient_loading: true
+  fsdp_offload_params: false
+  fsdp_sharding_strategy: FULL_SHARD
+  fsdp_state_dict_type: SHARDED_STATE_DICT
+  fsdp_sync_module_states: true
+  fsdp_transformer_layer_cls_to_wrap: LlamaDecoderLayer
```

Comment:
hmm I think its safest to unset it by default like Fabian did in his PR https://github.com/foundation-model-stack/fms-hf-tuning/pull/91/files#diff-7e2cb0844b4ef444122dc5020b67bfd949b38de258a1c63dbfa8abad0a746988R37

Reply:
@Ssukriti  Thank you, I have cherry picked this from Fabian's PR automatically credited him in this commit - https://github.com/foundation-model-stack/fms-hf-tuning/pull/92/commits/4aa0153ab0c55ac5b305b8d60e8f00f89cc354d4